lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -436,6 +438,13 @@ migrateManually tr defaultFieldValues =
             $ W.unActiveSlotCoefficient
             $ defaultActiveSlotCoefficient defaultFieldValues
 
+    -- | This table became @protocol_parameters@.
+    removeOldTxParametersTable :: Sqlite.Connection -> IO ()
+    removeOldTxParametersTable conn = do
+        dropTable <- Sqlite.prepare conn "DROP TABLE IF EXISTS tx_parameters;"
```

Comment:
Could be nice to first look for the table using `isFieldPresent` and then, log accordingly depending on the outcome?